spec.md

```diff
@@ -251,6 +251,8 @@ The following attributes are REQUIRED to be present in all CloudEvents:
 
 - Constraints:
   - REQUIRED
+  - MUST be a non-empty URI-reference
+  - RECOMMENDED that an absolute URI be used
```

Comment:
Disclaimer: I'm not a native English speaker. `be used` sounds strange to me here.
Either try `is used` or maybe `An absolute URI is RECOMMENDED`?

Reply:
works for me!